tests/data/README.md

```diff
@@ -0,0 +1,90 @@
+# Test images
+
+## Still images
+
+### Files `kodim*`
+
+![](io/kodim03_yuv420_8bpc.avif)
+
+![](io/kodim23_yuv420_8bpc.avif)
+
+License: released by the Eastman Kodak Company for unrestricted usage
+
+Source: <http://r0k.us/graphics/kodak/>
+
+### Files `cosmos*`
+
+![](io/cosmos1650_yuv444_10bpc_p3pq.avif)
+
+License: [Creative Commons Attribution license (reuse allowed)](https://creativecommons.org/licenses/by/3.0/legalcode)
+
+Source: ["Cosmos Laundromat" movie](https://www.youtube.com/watch?v=Y-rmzh0PI3c) (frame at around 1 minute past the
```

Comment:
It isn't critical, but I realized you could give a time offset here to bring the clip somewhat close to where you wanted it:
https://youtu.be/Y-rmzh0PI3c?t=60